kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MongoWindowedTable.java

```diff
@@ -171,6 +194,7 @@ public MongoWindowedTable(
     );
 
     this.database = client.getDatabase(name).withCodecRegistry(pojoCodecRegistry);
+    this.adminDatabase = client.getDatabase("admin");
     this.metadata = database.getCollection(
```

Comment:
We intentionally only shard the per-partition data collections, right? ie we don't need to shard the shared metadata collection?

Reply:
Now that you mention it I suppose we could. Seems simpler to just leave it unsharded in the sense that there are less possible configurations for it.